oxidation/client/package.json

```diff
@@ -15,8 +15,9 @@
     "android:open": "npm run android:copy && npx cap open android",
     "ios:open": "npm run ios:copy && npx cap open ios",
     "electron:open": "npm run electron:copy && npx cap open @capacitor-community/electron",
-    "electron:dist": "npx ionic build && npx cap copy @capacitor-community/electron && cd electron && npm run electron:dist"
-
+    "electron:dist": "npx ionic build && npx cap copy @capacitor-community/electron && cd electron && npm run electron:dist",
+    "wasm:build": "cd ../bindings/web && wasm-pack build -t web",
```

Comment:
can you use long option when writing script, it's more readable than a wild `-t` ;-)